features/endpoints/grafana/client/pom.xml

```diff
@@ -55,12 +55,12 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
-      <version>${okhttpVersion}</version>
+      <version>3.10.0</version>
```

Comment:
Use version [3.14.9](https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/3.14.9) which is latest on 3.x and it's dependent version  for okio [1.17.2](https://mvnrepository.com/artifact/com.squareup.okio/okio/1.17.2)

Declare these as `okhttp3Version` and  `okioGrafanaVersion` in original pom file

You have to re-target this branch to `release-33.x` 

We also need to add a smoke test with selenium to make sure this will not break again.